tests/src/Context/DrupalContext.php

```diff
@@ -75,4 +75,23 @@ public function assertFieldsNotPresent($fields) {
       }
     }
   }
+
+  /**
+   * Checks the users existence.
+   *
+   * @param string $username
+   *   The username of the user.
+   *
+   * @throws \Exception
+   *   Thrown when the user is not found.
+   *
+   * @Then I should have a :username user
```

Comment:
I think it is better to be "I should have an :username user" because the 'a' refers to the word 'user' so it is better as 'an' than 'a' (don't curse me too much.)